src/model/ControllerOutdoorAir.hpp

```diff
@@ -119,6 +119,9 @@ namespace model {
     boost::optional<std::string> getHeatRecoveryBypassControlType() const;
     bool setHeatRecoveryBypassControlType(const std::string& v);
 
+    boost::optional<std::string> getEconomizerOperationStaging() const;
+    bool setEconomizerOperationStaging(const std::string& v);
```

Comment:
Pffff. Another one where the existing API doesn't follow the convention.

This should be `std::string economizerOperationStaging() const`. Not getXXXX. And it has a default (at least currently), so no need to return an optional.
And there should be a `isEconomizerOperationStagingDefaulted` and a `resetEconomizerOperationStaging` if we do intend to have a default instead of making it required-field.

@kbenne thoughts please.

Reply:
It seems like we can keep (and depricate) the existing "get" method and add the new (correct) `std::string economizerOperationStaging() const`. Is that right?

Reply:
No deprecation necessary - this is a new field. I'll follow the convention for required-field (which is "correct" but inconsistent with other fields on this object).

Reply:
This specific comment was more about fixing the *existing* API, not the new field.
You don't have to do it here, we have lots to do at the moment. But we should file an issue to fix it later if it's not done here.

Reply:
https://github.com/NREL/OpenStudio/issues/4981